### DISCUSSION TITLE
Purevpn - port still closed?

### DISCUSSION THREAD

**Discussion by smdatt** [haugene/docker-transmission-openvpn#2870] [Q&A]
Hi, 

Not sure if this is a bug or just me misconfiguring. 

Container in K3s with nginx ingress. The pod is set to ClusterIP

I've set up PureVPN using my own config (one available when selecting port forwarding enabled configs on the PureVPN site). 

I've got the Port Forwarding addon enabled and set up for port 51413

The VPN is on and loaded successfully as I can see tun0 is set to the correct public IP as the config & the IP returned by `echo "Your IP: $(dig +short myip.opendns.com @resolver1.opendns.com)"` is also the public IP. 

Netstat shows that the ports are open - 
```
tcp        0      0 10.40.8.21:51413        0.0.0.0:*               LISTEN
tcp        0      0 0.0.0.0:9091            0.0.0.0:*               LISTEN
tcp6       0      0 :::51413                :::*                    LISTEN
```

In my config file, the env vars, and in the interface, the port value is 51413. 

However, in the Transmission GUI, it still shows `Port is closed` 

What am I missing?

**Comment by ghost** [comment]
Same issue for me with PureVPN and port forwarding option.
I haven't seen the port open since the end of 2023.
